data/human/hud outfits.txt

```diff
@@ -0,0 +1,87 @@
+# PARTICULAR PURPOSE. See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program. If not, see <https://www.gnu.org/licenses/>.
+
+outfit "In-flight Mass Display"
+	category "Heads Up Displays"
+	cost 5000
+	thumbnail "outfit/tactical scanner"
+	"mass" 0
+	"outfit space" 0
+	"mass display" 1
+	description "This innocuous device is so compact as to barely count as an outfit at all. Some pilots find it useful to know what their ship's current mass is while in-flight, and this little display adds that information to their Heads Up Display."
+
+outfit "Hyperjump Fuel Display"
+	category "Heads Up Displays"
+	cost 11000
+	thumbnail "outfit/tactical scanner"
+	"mass" 0
+	"outfit space" 0
+	"hyperdrive fuel cost display" 1
+	description "This innocuous device is so compact as to barely count as an outfit at all. Its sole purpose is to allow pilots to customize their heads-up display to provide their flight computer's current estimate of the amount of fuel they need to make a hyperjump."
+
+outfit "Scram drive Fuel Display"
+	category "Heads Up Displays"
+	cost 17000
+	thumbnail "outfit/tactical scanner"
+	"mass" 0
+	"outfit space" 0
+	"scram drive fuel cost display" 1
+	description "This innocuous device is so compact as to barely count as an outfit at all. Its sole purpose is to allow pilots to customize their heads-up display to provide their flight computer's current estimate of the amount of fuel they need to make a hyperjump with a scram drive."
+
+outfit "Jump drive Fuel Display"
+	category "Heads Up Displays"
+	cost 23000
+	thumbnail "outfit/tactical scanner"
+	"mass" 0
+	"outfit space" 0
+	"jump drive fuel cost display" 1
+	description "This innocuous device is so compact as to barely count as an outfit at all. Its sole purpose is to allow pilots to customize their heads-up display to provide their flight computer's current estimate of the amount of fuel they need to make a hyperjump with a jump drive."
+
+outfit "Fuel bar hyperjump display"
+	category "Heads Up Displays"
+	cost 7000
+	thumbnail "outfit/tactical scanner"
+	"mass" 0
+	"outfit space" 0
+	"hyperdrive fuel bar scale" 1
+	description "This innocuous device is so compact as to barely count as an outfit at all. Its sole purpose is to allow pilots to customize their heads-up display so that their fuel bar is segmented into sections representing a single hyperdrive jump."
+	description "	Just like the HUD display, the amount of fuel represented by each segment of the bar is automatically calculated in flight, and while landed it uses the ship's fully outfitted but otherwise empty mass. Given how standardized hyperdrives are, this is the default segmentation most ships use, and it will override other fuel segmentation customizations. Due to limitations of the display, if you have 31 or more hyper jumps worth of fuel, the display will show as a solid bar."
```

Comment:
> Due to limitations of the display, if you have [...]

This feels fourth-wall breaking, even though it is correct.
~~I don't suppose that's something possible to fix?~~

Reply:
The outfit descriptions could definitely use some work. I suspect that later on I'll rewrite these to be much more in-character.

That being said, yes and no. 
No: As things stand, it's a hard limit that at a certain point, more segments just turns the bar into a solid mass of breaks between segments with no segments themselves visible. That currently happens around 3300-3500 fuel supposedly, but I've seen problems with ships once they get past 3100 fuel. 

Yes: We could either reduce the separation between segments, that would allow a few more segments, but not many. We could also just make the bars longer. The longer the bar, the more fuel segments it can handle. The limit is the height of the screen. I'm not sure it would look good to have a fuel bar that extends the full height of the screen.